elf-size-analyze.py

```diff
@@ -1039,6 +1069,23 @@ def print_tree(header, symbols):
         for line in lines:
             line.print()
 
+    def create_json(header, symbols):
```

Comment:
When this is changed to just use `print` instead of opening a file (see comment below), we can rename it `print_json`.